packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol

```diff
@@ -275,6 +298,10 @@ contract BullStrategyTestFork is Test {
             usdcToBorrow
         );
         assertEq(IERC20(usdc).balanceOf(user1).sub(userUsdcBalanceBefore), usdcToBorrow);
+        assertTrue(
```

Comment:
if there ever a fuzz scenario that hits this assertion?  or is the cap too high relative to the crab balance user1 deposits in?

Reply:
Can we test an explicit revert for the cap being hit?  Doesn't have to be in a fuzz test.